lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,386 @@
+
+  - name: chargeitem.quantity
+    required: 'Yes'
+    type: quantity
+    description: Quantity of which the charge item has been serviced
+    action: create
+    example: |
+      {
+         "parameter": [
+            {
+              "name": "chargeItem",
+               "quantity": {
+                  "value": 1
+              }
+            }
+         ]
+      }
+    url: https://hl7.org/fhir/chargeitem-definitions.html#ChargeItem.quantity
+
+  - name: chargeitem.code
+    required: 'Yes'
+    type: CodeableConcept
+    description: A code that identifies the charge, like a billing code
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "chargeItem",
+            "code": {
+              "coding": [
+                {
+                  "extension": [
+                    {
+                      "valueCoding": {
+                        "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/14002",
+                        "code": "667687"
+                      },
+                      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                    }
+                  ],
+                  "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/CodeSystem/BillCodes-CDM_SCHED",
+                  "code": "01012021"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    url: https://hl7.org/fhir/chargeitem-definitions.html#ChargeItem.code
+
+  - name: chargeitem.performer
+    required: 'Yes'
+    type: BackboneElement
+    description: Who performed charged service
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "chargeItem",
+            "performer": [
+              {
+                "function": {
+                  "coding": [
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                      "code": "OP",
+                      "display": "Ordering Provider"
+                    }
+                  ],
+                  "text": "Ordering Provider"
+                },
+                "actor": {
+                  "reference": "Practitioner/190182805"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    url: https://hl7.org/fhir/chargeitem-definitions.html#ChargeItem.performer
+
+  - name: chargeitem.reason
+    required: 'Yes'
+    type: CodeableConcept
+    description: Why was the charged service rendered?
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "chargeItem",
+            "reason": [
+              {
+                "extension": [
+                  {
+                    "valueUnsignedInt": 1,
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+                  }
+                ],
```

Comment:
If this description is for reason field, I don't think so you need to add coding and extension field for this.
Any opinion @SantanuBanerjee ?

Reply:
[Fixed here](https://github.com/krvicky31/fhir.cerner.com/commit/155c91bc8fd4c872bcdff517e52e8bd3af56021d#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR366)